smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -364,7 +364,7 @@ protected async Task VerifyDataOnIoTHub(string moduleId)
                 EventPosition.FromEnd());
 
             var result = new TaskCompletionSource<bool>();
-            using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(10)))
+            using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(20))) // This long timeout is needed in case event hub is slow to process messages
```

Comment:
Please add TODO here, we should use the similar approach as what you did in long haul to verify test result instead of using events from event hub.